deselected_tests.yaml

```diff
@@ -382,7 +382,10 @@ deselected_tests:
 
   # Deselections for 2025.0
   - ensemble/tests/test_forest.py::test_importances[ExtraTreesRegressor-squared_error-float64]
-
+  - cluster/tests/test_k_means.py::test_kmeans_elkan_results[42-0.01-sparse-normal]
+  - cluster/tests/test_k_means.py::test_kmeans_elkan_results[42-1e-08-sparse-normal]
+  - cluster/tests/test_k_means.py::test_kmeans_elkan_results[42-0-sparse-normal]
+  - cluster/tests/test_k_means.py::test_kmeans_elkan_results[42-1e-100-sparse-normal]
```

Comment:
is this also reproduced on github actions? if not I would suggest revert this, since #2081 targeted to this

Reply:
Then we can merge his first, this will have a fully green CI, and we can then merge it into this PR.